docs/conf.py

```diff
@@ -455,8 +455,13 @@
     "notebooks/diagnostics/langmuir_analysis": (
         "_static/notebook_images/langmuir_analysis.png"
     ),
+    "notebooks/formulary/satellite-constellation": (
+        "_static/notebook_images/mms.png"  # public domain
+    ),
```

Comment:
```suggestion
    "notebooks/formulary/mms": (
        "_static/notebook_images/mms.png"
    ),  # public domain
```

This image was NOT being used for the thumbnail and this is probably why.  Additionally, `mms.ipynb` is not a very explicit name.  I'd choose something a little more descriptive...e.g. `earths_magnetosphere.ipynb`.